### DISCUSSION TITLE
Work with js files and flow

### DISCUSSION THREAD

**Discussion by krisidmisso** [microsoft/vscode-discussions#2388] [Extension Development QnA] [answered]
VS Code adds a red line under types: 
![image](https://github.com/user-attachments/assets/d932c013-0fb8-47df-a78e-8ce5713aa294)

Im working with js files. How to fix this lint issue?

**Comment by isidorn** [comment, marked as answer]
This forum is a place for vscode extension authors. For user queries it is best to ask them on stack overflow. Thank you
